doc/dnsseed-policy.md

```diff
@@ -44,13 +42,11 @@ details of their operating practices.
 related to the DNS seed operation.
 
 If these expectations cannot be satisfied the operator should
-discontinue providing services and contact the active Bitcoin
-Core development team as well as posting on
-[bitcoin-dev](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev).
+discontinue providing services and contact the active Zcash development team as well as creating an issue in the [Zcash repository](https://github.com/Zcash/zcash).
```

Comment:
@arcalinea Once this comment is addressed, I will ACK and merge - if this is for RC1 (I don't see a milestone or anything on the ticket).